lib/openc_bot/company_fetcher_bot.rb

```diff
@@ -114,19 +114,18 @@ def send_report(params)
       end
     end
 
-    def report_run_to_oc(params)
+    def report_run_to_analysis_app(params)
       bot_id = to_s.underscore
       run_params = params.slice!(RUN_ATTRIBUTES)
       run_params.merge!(bot_id: bot_id, bot_type: "external", git_commit: current_git_commit)
       run_params[:output] ||= params.to_s unless params.blank?
-      _http_post(OC_RUN_REPORT_URL, run: run_params)
+      _http_post(ANALYSIS_RUN_REPORT_URL, run: run_params)
```

Comment:
Because it holds something different. Leaving the old name would be confusing.

Zero impact. The constant isn't used outside of this module/repo is it? (unlike the method you're mentioning above)